SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/Utils/DataTypeUtilities.h

```diff
@@ -6,7 +6,7 @@
 
 #include "SchemaGenerator/TypeStructure.h"
 
-extern TMap<UClass*, FString> ClassToSchemaName;
+extern TMap<FString, FString> ClassToSchemaName;
```

Comment:
ClassNameToSchemaName / ClassPathToSchemaName ?

Reply:
Should be ClassPath if what's being stored is the ClassPath.